eslint.config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
You can delete this file, `eslintrc` is correct. This was a mistake in the content made by me :)